### ISSUE TITLE
CI part drops exception on a include statement

### ISSUE THREAD

**Issue by mcboman** [FountainJS/fountain-generator#41]
I'm getting the below exception when running the generator. Looks like it is the CI module. Tried not to choose a CI tool and it gave me the same exception.

running node version 4.4.6.

```
events.js:141
      throw er; // Unhandled 'error' event
      ^

TypeError: this.options.ci.includes is not a function
    at writing (C:\Users\Bosto\AppData\Roaming\npm\node_modules\generator-fountain-webapp\node_modules\generator-fountain-gulp\generators\ci\index.js:7:25)
    at Object.<anonymous> (C:\Users\Bosto\AppData\Roaming\npm\node_modules\generator-fountain-webapp\node_modules\yeoman-generator\lib\base.js:431:23)
    at C:\Users\Bosto\AppData\Roaming\npm\node_modules\generator-fountain-webapp\node_modules\run-async\index.js:26:25
    at C:\Users\Bosto\AppData\Roaming\npm\node_modules\generator-fountain-webapp\node_modules\run-async\index.js:25:19
    at C:\Users\Bosto\AppData\Roaming\npm\node_modules\generator-fountain-webapp\node_modules\yeoman-generator\lib\base.js:432:9
    at processImmediate [as _immediateCallback] (timers.js:383:17)

```

**Comment by zckrs** [comment]
Hi, 
Knowed issue fixed in the next release.  Or you can update to node 6

**Comment by mcboman** [comment]
Sorry for the trouble. Just upgraded to node 6.

**Comment by zckrs** [comment]
Just released v0.7.1 who fixed your issue
